### DISCUSSION TITLE
No module named 'pydantic_core._pydantic_core' using AWS Lambda

### DISCUSSION THREAD

**Discussion by sbsarav02** [pydantic/pydantic#6643] [Question] [answered]
My AWS lambda arch is of x86_64 and arm64. 
How to create my package to target the above architecture?
I am using Fastapi and trying to host helloworld applicaiton.

**Comment by ranmengtw** [comment]
Fastapi just released a [new version](https://github.com/tiangolo/fastapi/releases/tag/0.100.1) several hours ago. I was having the same issue in my lambda. Upgrade to that version worked for me now.

**Reply by dongruixiao** [reply]
It doesn't seem to work for me

**Comment by Kludex** [comment, marked as answer]
See https://github.com/pydantic/pydantic/issues/6557 for more information.
